assets/js/custom/ProgramSection.js

```diff
@@ -0,0 +1,28 @@
+export class ProgramSection {
+  constructor (programId) {
+    this.programId = programId
+  }
+
+  showCustomTranslation (customTranslationApi, callback) {
+    const decodedCookie = document.cookie
+      .split(';')
+      .map(v => v.split('='))
+      .reduce((acc, v) => {
+        acc[decodeURIComponent(v[0].trim())] = decodeURIComponent(v[1].trim())
+        return acc
+      }, {})
+
+    let usersLanugage = ''
+    if (decodedCookie.hl !== undefined) {
+      usersLanugage = decodedCookie.hl.substring(0, 2)
+    } else {
+      usersLanugage = document.documentElement.lang.substring(0, 2)
+    }
+
+    customTranslationApi.getCustomTranslation(
+      this.programId,
+      usersLanugage,
+      callback
```

Comment:
Is this implying a no-op error callback so when there are no defined translations, it will do nothing?

Reply:
Yeah, it just won't do anything if there is an error because most of the time it just returns a 404 not found if there isn't any custom translation found for the language

Reply:
That's a bit too implicit for me.  Can we change the name of this argument to notAvailableCallback

https://github.com/danielzhang130/Catroweb/blob/9349d70a6763ecb1324e096f407b3bbf2a71849d/assets/js/api/CustomTranslationApi.js#L8

and pass a empty function here, or some other way to make it clearer that this function doesn't ways replace with custom translation?

```
noop = () => {}
```

```suggestion
      callback,
      noop
```

Reply:
Added a noop function as a callback